src/org/usfirst/frc/team694/robot/commands/BITCOINCheckCommand.java

```diff
@@ -0,0 +1,29 @@
+package org.usfirst.frc.team694.robot.commands;
+
+import org.usfirst.frc.team694.robot.Robot;
+import org.usfirst.frc.team694.util.IRSensor;
+
+import edu.wpi.first.wpilibj.command.Command;
+import edu.wpi.first.wpilibj.command.InstantCommand;
+
+/**
+ *
+ */
+public class BITCOINCheckCommand extends InstantCommand {
+    public Command BITCOINCommand;
+    public BITCOINCheckCommand() {
+        super();
+        this.BITCOINCommand = new BITCOINCommand();
+        // Use requires() here to declare subsystem dependencies
+        // eg. requires(chassis);
+    }
+
+    // Called just before this Command runs the first time
+    protected void initialize() {
+        if (Robot.isBITCOINAutomation) {
+            if (IRSensor.isCubeDetected()) {
```

Comment:
Should this be using the IR sensor class directly, or should it be going through the Acquirer subsystem, or even Robot. In Rafael, we use the IR sensor port, but we only have one sensor. I think it would be a good idea to do this in a more Object Oriented way, with an IR sensor object in the Acquirer class, and methods in the Acquirer class to get that information, not accessing the IR sensor class directly. I would suggest you discuss this more with Adris.